.buildkite/node-pipeline.yml

```diff
@@ -25,7 +25,7 @@ steps:
         run: node-maze-runner
         use-aliases: true
         verbose: true
-        command: ["-e", "koa.feature", "-e", "koa-1x.feature", "-e", "webpack.feature"]
+        command: ["--bind-address=0.0.0.0", "-e", "koa.feature", "-e", "koa-1x.feature", "-e", "webpack.feature"]
```

Comment:
Just to note that this shouldn't be need in the next Maze Runner release (but no need to hold up the PR for that).